frontend/src/components/ui/button.ts

```diff
@@ -8,6 +9,8 @@ import { html, literal } from "lit/static-html.js";
 import { TailwindElement } from "@/classes/TailwindElement";
 import { tw } from "@/utils/tailwind";
 
+type Variant = "neutral" | "danger";
```

Comment:
Why remove the primary variant?

Reply:
Because it doesn’t really exist, not for this use case. The design for moving from an unselected to a selected state switches from what in Shoelace is two different variants, and to me it makes more sense to represent that as a progression from one state to another within a single variant — a button shouldn’t change what _kind_ of button it is depending on the state it’s in

Reply:
Hmm yeah let's continue this discussion with the button consolidation task. Maybe we don't need `btrix-button` and we just use `sl-button` with some custom styles.

Reply:
Actually, can we just use sl-button here? Then we only need to refactor one other case of btrix-button and remove the custom component altogether.

Reply:
Yeah probably, actually! I wish styling Shoelace elements was just a little easier with Tailwind, but that shouldn't be an issue. We can probably update the navigation button to use sl-button too, in that case, although maybe that's a situation where we keep the custom component just because of how often it's used and just change it to extend sl-button.

Reply:
To handle in https://github.com/webrecorder/browsertrix/issues/1550